README.md

```diff
@@ -93,8 +93,10 @@ resource_groups:
 resource_tags:
   - resource_tag_name: "group"
     resource_tag_value: "tomonitor"
+    resource_types:
+     - "Microsoft.Compute/storage"
     metrics:
-    - name: "CPU Credits Consumed"
+     - name: "CPU Credits Consumed"
```

Comment:
Why this extra space prefix (`resource_groups` example above do not have it)

Reply:
Same as above